src/Exception/BadRequestHttpException.php

```diff
@@ -0,0 +1,22 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Exception;
+
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException as BaseBadRequestHttpException;
+
+/**
+ * This exception is thrown when there is an issue with the request.
+ *
+ * @author Christian Schiffler <https://github.com/discordier>
+ */
+class BadRequestHttpException extends BaseBadRequestHttpException
```

Comment:
Eh, it is for every exception.
The omitted exceptions are only unknown exception classes implementing `HttpExceptionInterface` for which I naturally do not have a template and therefore can not handle them properly.
But these exceptions will get swapped with the 500-error screen, when pretty screens are switched on, in lack of another exception template See (https://github.com/contao/core-bundle/pull/70/files#diff-cdf99235fbbeccfa12764a4be188ae32R114).

Affected (known) exceptions currently are:
```php
\Symfony\Component\HttpKernel\Exception\HttpException
\Symfony\Component\HttpKernel\Exception\BadRequestHttpException
\Symfony\Component\HttpKernel\Exception\UnsupportedMediaTypeHttpException
\Symfony\Component\HttpKernel\Exception\ConflictHttpException
\Symfony\Component\HttpKernel\Exception\MethodNotAllowedHttpException
\Symfony\Component\HttpKernel\Exception\UnauthorizedHttpException
\Symfony\Component\HttpKernel\Exception\TooManyRequestsHttpException
\Symfony\Component\HttpKernel\Exception\NotAcceptableHttpException
\Symfony\Component\HttpKernel\Exception\UnprocessableEntityHttpException
\Symfony\Component\HttpKernel\Exception\PreconditionFailedHttpException
\Symfony\Component\HttpKernel\Exception\ServiceUnavailableHttpException
\Symfony\Component\HttpKernel\Exception\PreconditionRequiredHttpException
\Symfony\Component\HttpKernel\Exception\GoneHttpException
\Symfony\Component\HttpKernel\Exception\NotFoundHttpException
\Symfony\Component\HttpKernel\Exception\AccessDeniedHttpException
\Symfony\Component\HttpKernel\Exception\LengthRequiredHttpException
```